### PR TITLE
[feat] Indent issue body when working with multi-issue file

### LINKED ISSUE #13: Reformat issue confirmation output on using Multi Issue File
Right now when opening issues repobee asks for confirmation on these and the way the text is formatted doesn't really do it for my eyes.
```
wagmode$ repobee issues feedback --mi issues.md -a task-15
Processing issue "Pass" for student1-task-15: Now this is c++!

dolor sit amet
Open issue "Pass" in repo student1-task-15? (y/n) y
Processing issue "Fail" for student2-task-15: 
Open issue "Fail" in repo student2-task-15? (y/n) y
Processing issue "Pass" for student3-task-15: more lorem ipsum, 

more dolor sit amet
Open issue "Pass" in repo student3-task-15? (y/n) 
````
I think this feels a bit cluttered, especially when each issue is given one by one it is sometimes difficult to directly see what is the new/old issue. Instead I would want something like this:

```
wagmode$ repobee issues feedback --mi issues.md -a task-15

Processing issue "Pass" for student1-task-15: 
    >Now this is c++!
    >
    >lorem ispum dolor sit amet
Open issue "Pass" in repo student1-task-15? (y/n) y

Processing issue "Fail" for student2-task-15: 
Open issue "Fail" in repo student2-task-15? (y/n) y

Processing issue "Pass" for student3-task-15: 
    >More Lorem isum
    >
    >More dolor sit amet
Open issue "Pass" in repo student3-task-15? (y/n) 
```

So the changes i can think of right now is:
* Separate each issue with empty line
* Indent issue text and/or place in some kind of block

This becomes even more usefull for larger issues. 

Alternatively, we can also make it so that when a new issue is shown in the terminal we make it so that while in "approval mode" only the current issue is shown. @slarse  what do you think?

### PR DESCRIPTION
**Pull request by marceloFA**
Fix #13

I opted to not implement terminal cleaning for each issue because of the scrolling up little problem. If you wish to have this feature (cleaning the terminal for each issue read), I can implement it, no problem :)

Please comment on anything that does not look good for you, I'm always totally open to suggestions and changes :D

### CODE REVIEW

**Comment by marceloFA** [comment]
I should have setup pre-commit before submitting this. had to do an extra commit and learned the lesson haha

File: repobee_feedback/feedback.py [hunk 1]
```diff
@@ -110,11 +112,10 @@ def command(self, api: plug.PlatformAPI):
 
 def _ask_for_open(issue: plug.Issue, repo_name: str, trunc_len: int) -> bool:
     plug.echo(
-        'Processing issue "{}" for {}: {}{}'.format(
+        '\nProcessing issue "{}" for {}:\n{}'.format(
             issue.title,
             repo_name,
-            issue.body[:trunc_len],
-            "[...]" if len(issue.body) > trunc_len else "",
```
Comment (slarse) [inline repobee_feedback/feedback.py:117]: The `[...]` shouldn't be removed. It gives an indication that there's more to see when the body of the issue is truncated. There's no test for this, which is a bit unfortunate, as it's a key part of the user interface of this plugin.

File: repobee_feedback/feedback.py [hunk 2]
```diff
@@ -11,13 +11,15 @@
...
 import argparse
+from textwrap import indent
 from typing import Iterable, Tuple, List, Mapping
 
 import repobee_plug as plug
 
 PLUGIN_NAME = "feedback"
 
 BEGIN_ISSUE_PATTERN = r"#ISSUE#(.*?)#(.*)"
+INDENTATION_STR = "    >"
```
Comment (slarse) [inline repobee_feedback/feedback.py:22]: After looking at it, I don't think I'm too crazy about the `>`, especially as it has meaning in Markdown (which matters here, as issue bodies are formatted with Markdown on all supported platforms). How about we just go with pure indentation (spaces or a single tab)?

**Review by slarse — CHANGES_REQUESTED** [review]
Hi @marceloFA, thanks for your PR!

I have a few comments on the code changes.

Also, while we're already poking about here, I figured we can refactor the `_ask_for_open` function to use f-strings. Something like this:

```python
def _ask_for_open(issue: plug.Issue, repo_name: str, trunc_len: int) -> bool:
    indented_body = indent(issue.body[:trunc_len], INDENTATION_STR)
    issue_description = (
        f'\nProcessing issue "{issue.title}" for {repo_name}:\n{indented_body}'
    )
    plug.echo(issue_description)
    return (
        input(f'Open issue "{issue.title}" in repo {repo_name}? (y/n) ') == "y"
    )
```

And then you need to work in that missing continuation indicator (`[...]`) somewhere as well.  What do you think?

If you're feeling extra productive, you could even extract the formatting of the issue description into a separate function, and add a couple of tests for the output.

**Comment by slarse** [comment]
I'm also a bit confused as to why the GitHub workflows haven't executed (or asked me to confirm a run, as is typically required for first-time contributors). Gonna have to look at that.

File: repobee_feedback/feedback.py (hunk 1 above)
Comment (marceloFA) [reply repobee_feedback/feedback.py:117]: Oooh, how could I have removed this? probably by mistake haha. Sure, I added it back

File: repobee_feedback/feedback.py (hunk 2 above)
Comment (marceloFA) [reply repobee_feedback/feedback.py]: Sure!

**Comment by marceloFA** [comment]
Added some unit tests and did some refactor work. Let me know what you think! :)

**Review by slarse — CHANGES_REQUESTED** [review]
Looks great! I just have some minor comments on the tests and then I think we're good to merge.

File: tests/test_feedback.py [hunk 3]
```diff
@@ -223,3 +223,23 @@ def test_skips_unexpected_issues_in_multi_issues_file(
...
+class TestIndentIssueBody:
+    """Tests for the method that addds indentation to the issue body"""
+
+    def test_issue_indented_and_truncated(
+        self,
+    ):
+        """Test that a long issue body get truncated to a certain length"""
+        body = "Hello\nworld\n"
+        indented_body = feedback._indent_issue_body(body, 10)
+        assert feedback.TRUNC_SIGN in indented_body
```
[the remark this answers is not in the document]
Comment (marceloFA) [reply tests/test_feedback.py:237]: Sure! not using magic numbers is a good idea, thanks for the suggestions.

**Review by slarse — APPROVED** [review]
Thanks @marceloFA, looks great!

Still have no idea why the tests aren't running, but I ran them locally and it's all good.

**Pull request merged**